R/app.R

```diff
@@ -1,3 +1,5 @@
+app_version <- "0.5.3"
```

Comment:
I don't think this object works here. I questioned it so I forked your branch on my local machine and it gave me an error. If you move the object inside R/ui.R, then it should actually work. 

I had worked for you because you had loaded in your local environment the object, so it wasn't giving you an error. But it did not work for me because I  did not have the variable in my environment.